apps/gallery/test/marionette/lib/gallery.js

```diff
@@ -43,7 +44,15 @@ Gallery.Selector = Object.freeze({
   editToolApplyButton: '#edit-tool-apply-button',
   editHeader: '#edit-view gaia-header',
   fullscreenFrame2: '#frame2',
-  fullscreenFrame3: '#frame3'
+  fullscreenFrame3: '#frame3',
+  cropDoneButton: '#crop-done-button',
+  editCropCanvas: '#edit-crop-canvas',
+  actionMenu: 'form[data-z-index-level="action-menu"]',
+  open: {
+      title: '#filename',
+      image: '#frame > div.image-view',
+      saveButton: '#save'
+    }
```

Comment:
I think it might be better to create a separate Open.Selector object to hold these rather than nesting them like this.  Or keep them here but prefix the property names with open?  openTitle, openImage, etc.? 

I say this because if someone was going to create a utility that would automatically generate a getter method for every selector, you'd want an object that was only one level deep...